app/action/real-time-client-action.coffee

```diff
@@ -52,28 +52,35 @@ parseMessage = (topic, message, actionContext) ->
 module.exports =
   startRealTimeClient: (actionContext, options, done) ->
     #Fetch initial data
-    xhrPromise.getJson(config.URL.REALTIME + (getTopic(options)).replace('#', '')).then (data) ->
-      parseMessage(topic, message, actionContext) for topic, message of data
+    if !Array.isArray(options)
```

Comment:
Should we restrict that options is always array. Now this makes me think: "why it is called in different ways"